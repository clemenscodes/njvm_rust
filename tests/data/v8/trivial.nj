// Simple program to test things quickly

void main() {
    if (readInteger() == 42) {
        writeString("Exactly!\n");
    } else {
        writeString("Nope!\n");
    }
}