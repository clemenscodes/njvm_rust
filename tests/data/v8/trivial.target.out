Ninja Virtual Machine started
Exactly!
Ninja Virtual Machine stopped
